admin/scripts/modules/uvod.php

```diff
@@ -106,19 +100,6 @@
   $x->assign('ok', $ok='<img src="files/design/ok-s.png" style="margin-bottom:-2px">');
   $x->assign('err',$err='<img src="files/design/error-s.png" style="margin-bottom:-2px">');
   $a=$up->koncA();
-  $novacci=array_map(function($n){
-    global $ok, $err;
-    $a=$n->koncA()?$n->koncA().',':',&ensp;';
-    return 
-      $n->jmenoNick()
-      .'&emsp;<form method="post" class="radkovy"><input type="submit" value="zrušit"><input type="hidden" name="zrusitNovacka" value="'.$n->id().'"></form>'
-      .'<ul><li>'
-      .($n->gcPrihlasen() ? $ok.' přihlášen'.$a : $err.' nepřihlášen'.$a )
-      .($n->gcPritomen()  ? $ok.' dorazil'.$a   : $err.' nedorazil'.$a )
-      .($n->finance()->stav()<0 ? $err : $ok)
-      .' '.$n->finance()->stavHr().''
-      .'</ul></li>';
-  },$up->novacci());
```

Comment:
Metoda Uzivatel::novacci() už se nikde nepoužívá, a měla by se tedy smazat. To platí i pro sloupec uzivatele_hodnoty.guru v databázi.